Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.16 AS builder
+FROM registry.ci.openshift.org/openshift/release:golang-1.17 AS builder
 WORKDIR /go/src/github.com/openshift/machine-api-provider-powervs
 COPY . .
 # VERSION env gets set in the openshift/release image and refers to the golang version, which interfers with our own
 RUN unset VERSION \
  && GOPROXY=off NO_DOCKER=1 GOARCH=ppc64le make build
 
-FROM --platform=ppc64le registry.access.redhat.com/ubi8/ubi:8.4
+FROM --platform=ppc64le registry.ci.openshift.org/openshift/origin-v4.0:base
```

Comment:
lets keep this as is, the modified image is not a multi-arch image
```shell
% manifest-tool inspect registry.ci.openshift.org/openshift/origin-v4.0:base
Name: registry.ci.openshift.org/openshift/origin-v4.0:base (Type: application/vnd.docker.distribution.manifest.v2+json)
      Digest: sha256:cbd0e2931e6ae8cbd1522cd7226cff89629c21a3652f6fd57d84f405d432c07a
        Size: 1155
          OS: linux
        Arch: amd64
    # Layers: 4
      layer 01: digest = sha256:c2340472a0fa0c6c0b7910b6c292f627448da15d5e3c375c61c3141f494a3268
      layer 02: digest = sha256:6e55351c18ffebc0918b4c21c1257d28d7e311bd37009fb722cb59761b7449ed
      layer 03: digest = sha256:b2d2704dda6c98b6a775e8e5566af7a92f700b542905bd94310dc18b9bc2d0b0
      layer 04: digest = sha256:a2c10be042b935d0f2657bca65ddc5b52f542501ad20d6b29c2558352f723837
```

Reply:
Reverted back the image in the latest commit